app/views/docs/oauth-providers/github.md

```diff
@@ -0,0 +1,348 @@
+# GitHub provider
+
+Appwrite allows you to authenticate users using their GitHub account through the GitHub OAuth2 provider. OAuth authentication is a great way to reduce friction for your users and increase user conversion by simplifying the signup process. You can learn more about Appwrite's other OAuth2 providers [here](placeholder link). 
+
+## Enabling the GitHub provider
+Before you can use GitHub to authenticate users, you need to enable the provider in your Appwrite console.
+1. Navigate to your Appwrite project
+2. Navigate to **Auth** > **Settings**
+3. Find and open the OAuth provider
+4. In the **Github OAuth2 Settings** modal, use the toggle to enable the provider
```

Comment:
```suggestion
4. In the **GitHub OAuth2 Settings** modal, use the toggle to enable the provider
```

This is going to be fixed in 1.4.x:

![image](https://github.com/appwrite/docs/assets/1477010/204b722c-c5aa-43a4-85a8-529cdb4a292b)